.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "7"
+  - "8"
```

Comment:
Let's keep latest only so that tests (when we have proper ones) don't run twice.  

We could probably even get rid of iojs.

Reply:
Yeah wasn't sure yet, that's why I left it open. But I believe we then can just use `node_js: node` to use the latest version